### ISSUE TITLE
Add feature : export/import config

### ISSUE THREAD

**Issue by Alwaysin** [FreshRSS/FreshRSS#851]
I was wondering if it was possible to export and import the config, using the same principle than when exporting feeds. The usefulness might be limited but when changing instances it would save time!

What do you think about it?

**Comment by Alkarex** [comment]
Yes, we could make something more user-friendly, but for the time being, it is already possible to save `data/config.php` and/or the users' configuration files `data/users/*/config.php`

**Comment by Alkarex** [comment]
See also https://github.com/FreshRSS/FreshRSS/issues/890

**Comment by math-GH** [comment]
#4553
